include/triton/Dialect/TritonGPU/IR/LinearLayoutConversions.h

```diff
@@ -282,6 +282,11 @@ LinearLayout chooseScaledMfmaScaleLayout(
     MLIRContext *ctx, int dotOperandIdx,
     const std::vector<std::vector<int32_t>> &dotOperandWarpBasis,
     ArrayRef<int64_t> dotOperandShape, unsigned mfmaMDim);
-} // namespace mlir::triton::gpu
 
+// Create a LinearLayuot similar to mfmaLayout where each thread holds 8
```

Comment:
Typo in "LinearLayuot". ".. mfmaLayout but changing each thread to hold.. Such layout is useful for emitting the widest 128-bit global store instructions."

Reply:
Yi, you need to emphasize that this chosen layout is "similar" to the mfma layout in a way that the layout conversion between them can be done with transferWithinWarp, i.e. no LDS is needed.